src/app/public/modules/input-box/input-box.component.ts

```diff
@@ -36,9 +40,20 @@ import {
 })
 export class SkyInputBoxComponent implements OnInit {
 
+  /**
+   * Indicates whether to display the input box in an error state. The input box component will
```

Comment:
Same as the `disabled` state. I'd defer to @Blackbaud-PaulCrowder on this as well.

Reply:
There shouldn't be any special styling applied to the `input` element when in an error state, so it would only go on the input box itself.

Reply:
Got it.
@blackbaud-johnly Yeah, I'd say there is a functional distinction here between display and actual error states. The input box component will automatically display the input box in an error state if the input element is wired up to an Angular reactive form or `ngModel`. However, if the consumer wanted to show an error state without wiring up the input element, then this boolean would allow them to do so.
Any suggestions on how to best describe that?

Reply:
What do you think of:

"Indicates whether to visually highlight an error state for the input box even if no error exists for the `input` element."

Reply:
I like that, but should we still let the consumer know that it will automatically highlight if the error state exists? Or do you think its implied in this sentence?

Reply:
> Got it.
> @blackbaud-johnly Yeah, I'd say there is a functional distinction here between display and actual error states. The input box component will automatically display the input box in an error state if the input element is wired up to an Angular reactive form or `ngModel`. However, if the consumer wanted to show an error state without wiring up the input element, then this boolean would allow them to do so.
> Any suggestions on how to best describe that?

Calling out this scenario seems like it would cause more confusion than leaving it out. I'd prefer leaving the wording something like "indicates the input is in an error state."